app/src/main/res/drawable/redcurve.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
the title of this file should be more descriptive and use underscores to separate words

Reply:
done